src/input_common/sdl/sdl_impl.cpp

```diff
@@ -95,6 +96,10 @@ class SDLJoystick {
         return std::make_tuple(x, y);
     }
 
+    InputCommon::MotionInput GetMotion() const {
```

Comment:
```suggestion
    const InputCommon::MotionInput& GetMotion() const {
```

MotionInput instances are expensive to copy, so its best to allow the calling code to take it by reference if it only needs to query information.